### PR TITLE
Update PMDARIMA examples in docstrings

### PR DESCRIPTION
**Pull request by michael-berk**
## Related Issues/PRs

 #7501

## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested?

- [] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests (describe details, including test results, below)

`make clean; make rsthml` and observe output. I also ran each of these code blocks in a Databricks notebook. 

## Does this PR change the documentation?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Make sure the changed pages / sections render correctly in the documentation preview.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [x] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/pmdarima.py [hunk 1]
```diff
@@ -310,7 +424,69 @@ def load_model(model_uri, dst_path=None):
...
+
+        # Forecast for the next 60 days
+        forecast = loaded_model.predict(n_periods=60)
+
+        print(f"forecast: {forecast}")
+
+    .. code-block:: text
+        :caption: Output
+
+        Model artifact logged to: dbfs:/databricks/mlflow-tracking/969987237801842/
```
Comment (BenWilson2) [inline mlflow/pmdarima.py:480]: Let's remove this to make it OSS-friendly :)

File: mlflow/pmdarima.py [hunk 2]
```diff
@@ -310,7 +424,69 @@ def load_model(model_uri, dst_path=None):
...
+            input_sample = pd.DataFrame(train["sales"])
+            output_sample = pd.DataFrame(model.predict(n_periods=5))
+            signature = infer_signature(input_sample, output_sample)
+
+            # Log model
+            mlflow.pmdarima.log_model(model, ARTIFACT_PATH, signature=signature)
+
+            # Get the model URI for loading
+            model_uri = mlflow.get_artifact_uri(ARTIFACT_PATH)
+            print(f"Model artifact logged to: {model_uri}")
```
Comment (BenWilson2) [inline mlflow/pmdarima.py:467]: Let's remove this print statement

File: mlflow/pmdarima.py [hunk 3]
```diff
@@ -310,7 +424,69 @@ def load_model(model_uri, dst_path=None):
...
+            prediction = model.predict(n_periods=len(test))
+            metrics = {"smape": smape(test["sales"], prediction)}
+
+            # Infer signature
+            input_sample = pd.DataFrame(train["sales"])
+            output_sample = pd.DataFrame(model.predict(n_periods=5))
+            signature = infer_signature(input_sample, output_sample)
+
+            # Log model
+            mlflow.pmdarima.log_model(model, ARTIFACT_PATH, signature=signature)
```
[the remark this answers is not in the document]
Comment (mberk06) [reply mlflow/pmdarima.py:463]: `input_sample`? If so, what's the purpose?

File: mlflow/pmdarima.py [hunk 4]
```diff
@@ -343,18 +519,7 @@ def __init__(self, pmdarima_model):
         self.pmdarima_model = pmdarima_model
         self._pmdarima_version = pmdarima.__version__
 
-    def predict(
-        self, dataframe, params: Optional[Dict[str, Any]] = None
-    ) -> pd.DataFrame:  # pylint: disable=unused-argument
-        """
```
Comment (BenWilson2) [inline mlflow/pmdarima.py:349]: Can we revert the deletion / modification of this?

**Review by BenWilson2 — APPROVED** [review]
LGTM! Thanks for the add :)

**Pull request merged**